DEVELOPMENT.md

```diff
@@ -256,6 +256,27 @@ The recommended minimum development configuration is:
   - 8 GB of (actual or virtualized) platform memory
 - Node autoscaling, up to 3 nodes
 
+#### Using [KinD](https://kind.sigs.k8s.io/)
+
+[KinD](https://kind.sigs.k8s.io/) is a great tool for working with Kubernetes clusters locally. It is particularly useful to quickly test code against different cluster [configurations](https://kind.sigs.k8s.io/docs/user/quick-start/#advanced).
+
+1. Install [required tools](./DEVELOPMENT.md#install-tools) (note: may require a newer version of Go).
+2. Install [Docker](https://www.docker.com/get-started).
+3. Create cluster:
+   
+   ```sh
+   $ kind create cluster
+   ```
+
+4. Configure [ko](https://kind.sigs.k8s.io/):
+   
+   ```sh
+   $ export KO_DOCKER_REPO="kind.local"
+   $ export KIND_CLUSTER_NAME="kind"  # only needed if you used a custom name in the previous step
```

Comment:
I don't remember, if the cluster name is different, shouldn't `KO_DOCKER_REPO` be different too ?

Reply:
Nope! Only KIND_CLUSTER_NAME needs to be set. 🧙 

I use this all the time having separate kind clusters for vanilla pipelines and pipelines + results.